src/language/__tests__/lexer-test.js

```diff
@@ -262,6 +262,95 @@ describe('Lexer', () => {
       end: 34,
       value: 'unicode \u1234\u5678\u90AB\uCDEF',
     });
+
+    expect(lexOne('"unicode \\u{1234}\\u{5678}\\u{90AB}\\u{CDEF}"')).to.contain(
+      {
+        kind: TokenKind.STRING,
+        start: 0,
+        end: 42,
+        value: 'unicode \u1234\u5678\u90AB\uCDEF',
+      },
+    );
+
+    expect(
+      lexOne('"string with unicode escape outside BMP \\u{1F600}"'),
+    ).to.contain({
+      kind: TokenKind.STRING,
+      start: 0,
+      end: 50,
+      value: 'string with unicode escape outside BMP 😀',
```

Comment:
I don't trust Unicode symbols in tests, because you can't visually inspect that codes are matching.
Can we use JS escape sequences instead?

Reply:
Actually, I wrote a simple ESLint rule to find non-ASCII characters in #3053
And discovered that my editor (NeoVim) use strange encoding for ⏱️ emoji:
![image](https://user-images.githubusercontent.com/8336157/117135322-73495580-adaf-11eb-93f3-49652b25086c.png)